features/src/FeatureContext.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Behat\MinkExtension\Tests;
+
+use Behat\Mink\Exception\ExpectationException;
+use Behat\MinkExtension\Context\RawMinkContext;
+
+/**
+ * Feature context for testing advanced scenarios.
+ */
+class FeatureContext extends RawMinkContext
+{
+
+    /**
+     * @BeforeScenario @MockXdebug
+     */
+    public function setUpXdebugMock()
+    {
+        $_SERVER['XDEBUG_CONFIG'] = 'xdebug';
+    }
+
+    /**
+     * @Then /^I should have the "([^"]*)" cookie with value "([^"]*)"$/
+     */
+    public function iShouldHaveTheCookieWithValue($cookie_name, $cookie_expected_value) {
```

Comment:
please respect our coding standards. We use PSR-2, so variable names are camelCased, and the curly brace should go on the next line (as done for the previous method)